internal/pkg/state/remote/manager.go

```diff
@@ -288,8 +308,13 @@ func (u *UnitOfWork) createOrUpdate(objectState model.ObjectState, object model.
 	setMetadata := !exists || changedFields.Has("metadata")
 	var setMetadataReq *client.Request
 	if setMetadata {
-		changedFields.Remove("metadata")
 		setMetadataReq = u.api.AppendMetadataRequest(object)
+		changedFields.Remove("metadata")
+		// If there is no other change, send the request and return
+		if len(changedFields) == 0 {
+			setMetadataReq.Send()
+			return nil
+		}
```

Comment:
Tady byl zřejmě bug. Pokud se změní jen metadata a nic jinýho, tak to pokračovalo dál do `u.updateRequest()` s prázdným `changedFields` a na tom mi failoval ten request do Storage, že měl prázdný body.

Reply:
:+1: